hazelcast-sql-core/src/test/java/com/hazelcast/sql/misc/SqlNoDeserializationTest.java

```diff
@@ -207,8 +208,17 @@ private void populate(boolean fail) {
             localMap.put(new PersonKey(i, fail), new Person(fail));
         }
 
-        member1.getMap(MAP_NAME).clear();
-        member1.getMap(MAP_NAME).putAll(localMap);
+        IMap<PersonKey, Person> map = member1.getMap(MAP_NAME);
+
+        map.putAll(localMap);
+
+        if (fail) {
+            for (int i = 0; i < KEY_COUNT; i++) {
+                map.remove(new PersonKey(i, false));
+            }
```

Comment:
Test seems timing sensitive, if i insert, for instance 300 millis here, it fails.
 
```suggestion
            }
  sleepAtLeastMillis(300);

```

Reply:
The timing problem could be caused by a periodic plan checker that attempts to invalidate suspicious plans. 
To make the test more reliable, I decided to rework it in favor of `Portable` objects. In SQL, we may need to deserialize the object in two cases: when building the schema, and when reading values from the user cursor. The goal of this test is to ensure that we do not do deserailziations in any other case, such as when transferring data between processes. If the object is `Portable`, we do not deserialize it when building the schema, because in this case, we extract fields directly from `Data`. Therefore, the change to the portable, removes exceptions during schema build, making the tests stable.